de.metas.document.archive/de.metas.document.archive.base/src/main/java/de/metas/document/archive/spi/impl/DocOutboundArchiveEventListener.java

```diff
@@ -98,7 +124,7 @@ protected I_C_Doc_Outbound_Log_Line createLogLine(final I_AD_Archive archive)
 		final Properties ctx = InterfaceWrapperHelper.getCtx(archive);
 		final String trxName = InterfaceWrapperHelper.getTrxName(archive);
 
-		I_C_Doc_Outbound_Log docExchange = Services.get(IArchiveDAO.class).retrieveLog(archive);
+		I_C_Doc_Outbound_Log docExchange = Services.get(de.metas.document.archive.api.IArchiveDAO.class).retrieveLog(archive);
```

Comment:
ok, i see..sins of the past..
can you please rename the one from `de.metas.document.archive.api` (and its impl of course) to something like maybe `(I)DocOutboundDAO` ?